README.md

```diff
@@ -95,6 +95,37 @@ The benchmark assumes that Vivaldi uses bottom navigation, available in settings
+Since the purpose of the benchmark is not to test some OEM cheating software you should perform a quick sanity check if the performance of your browser roughly falls roughly in line with other devices with the same chipset or primary CPU core. 
+The benchmark [Octane JS by Google](https://chromium.github.io/octane) is a pure Javascript benchmark and runs in less than a minute and yields very constistant results. If the results for your device are more than 30% off its likley that some manipulation is going on. You can also try to prevent this by enabling any performance mode and disabling any battery saving related features on the device if available, however in the case of Samsung and Oneplus this still did not remove the artifical limits.
+
+Also due to the singlethreaded nature of Javascript when using an App to monitor your phones CPU Clocks Speeds like [CPU Float](https://play.google.com/store/apps/details?id=com.waterdaaan.cpufloat&utm_source=global_co&utm_medium=prtnr&utm_content=Mar2515&utm_campaign=PartBadge&pcampaignid=MKT-Other-global-all-co-prtnr-py-PartBadge-Mar2515-1) upon starting the benchmarks the phone should almost immediatly reach the advertised max. CPU Clock or something close to it on at least one core.
+
+
+
+| Soc            | Primary CPU                    | OctaneJS Chromium | OctaneJS Firefox |
+|:--------------:|:------------------------------:|:-----------------:|:----------------:|
+| Snapdragon 810 | Cortex-A57 @ 2 GHz             | ~ 9500            | ~ 5400           |
+| Snapdragon 820 | Kryo @ 2.15 GHz                | ~ 11000           | ~ 6800           |
+| Snapdragon 835 | Modified Cortex-A73 @ 2.45 GHz | ~ 14500           | ~ 8700           |
+| Snapdragon 845 | Modified Cortex-A75 @ 2.8 GHz  | ~ 23000           | ~ 13000          |
+| Snapdragon 855 | Modified Cortex-A76 @ 2.84 Ghz | ~ 32000           | ~ 19000          |
+| Snapdragon 865 | Cortex-A77 @ 2.84 GHz          | ~ 41000           | ~ 22000          |
+| Exynos 7420    | Cortex-A57 @ 2.1 GHz           | ~ 10800           | ~ 5300           |
+| Exynos 8890    | Samsung M1 @ 2.6 Ghz           | ~ 13000           | ~ 8500           |
+| Exynos 8895    | Samsung M2 @ 2.3 Ghz           | ~ 17000           | ~ 9300           |
+| Exynos 9810    | Samsung M3 @ 2.9 Ghz           | ~ 24000           | ~ 12000          |
+| Exynos 9820    | Samsung M4 @ 2.7 Ghz           | ~ 33000           | ~ 18000          |
+| Kirin 955      | Cortex A-72 @ 2.55 Ghz         | ~ 14500           | ~ 8000           |
+| Kirin 960      | Cortex A-73 @ 2.3GHz           | ~ 13500           | ~ 8000           |
+| Kirin 970      | Cortex A-73 @ 2.3GHz           | ~ 14000           | ~ 8500           |
+| Kirin 980      | Cortex A-76 @ 2.6 Ghz          | ~ 31000           | ~ 17000          |
+| MTK Helio X20  | Cortex A-72 @ 2.3 Ghz          | ~ 11000           | ~ 6500           |
+
```

Comment:
Consider wrapping this in [spoiler text](https://gist.github.com/GuillaumeFalourd/9305b6372bc1ff7855004dcc83b6ff1d) to avoid an overlong README.